### PR TITLE
FS-2003-add-user-info-for-scores

### PR DESCRIPTION
**Pull request by gio-karunakaran**
adding user info for scores, user info for comments will be added as part of fs-772

### CODE REVIEW

File: config/envs/development.py [hunk 1]
```diff
@@ -8,11 +10,29 @@
...
+    # RSA 256 KEYS
+    RSA256_PUBLIC_KEY_BASE64 = getenv("RSA256_PUBLIC_KEY_BASE64")
+    if RSA256_PUBLIC_KEY_BASE64:
+        RSA256_PUBLIC_KEY = base64.b64decode(RSA256_PUBLIC_KEY_BASE64).decode()
+    if not hasattr(DefaultConfig, "RSA256_PUBLIC_KEY"):
+        _test_public_key_path = (
+            DefaultConfig.FLASK_ROOT + "/tests/keys/rsa256/public.pem"
+        )
+        with open(_test_public_key_path, mode="rb") as public_key_file:
+            RSA256_PUBLIC_KEY = public_key_file.read()
```
Comment (tferns) [inline config/envs/development.py:38]: Out of curiosity, where did this come from?

File: config/envs/unit_test.py [hunk 2]
```diff
@@ -6,3 +9,30 @@
...
+
+    RSA256_PUBLIC_KEY_BASE64 = getenv("RSA256_PUBLIC_KEY_BASE64")
+    if RSA256_PUBLIC_KEY_BASE64:
+        RSA256_PUBLIC_KEY = base64.b64decode(RSA256_PUBLIC_KEY_BASE64).decode()
+    if not hasattr(DefaultConfig, "RSA256_PUBLIC_KEY"):
+        _test_public_key_path = (
+            DefaultConfig.FLASK_ROOT + "/tests/keys/rsa256/public.pem"
+        )
+        with open(_test_public_key_path, mode="rb") as public_key_file:
+            RSA256_PUBLIC_KEY = public_key_file.read()
```
Comment (tferns) [inline config/envs/unit_test.py:38]: Same question for this

**Review by tferns — APPROVED** [review]
LGTM

File: config/envs/unit_test.py (hunk 2 above)
Comment (gio-karunakaran) [reply config/envs/unit_test.py:38]: as with in development config, these envvars are needed with the login_required decorators as it useses these to validate users and the tokens.

**Pull request merged**